### PR TITLE
Adjust transaction list headers (#370)

### LINKED ISSUE #370: Adjust transaction list headers
The entries of the transaction list will only contain the time, therefore the header between entries should correspond to the data (no grouping of multiple dates).

See https://github.com/gnosis/safe/blob/master/specs/transaction_list/02_user_interface.rst#1transaction-list

### PR DESCRIPTION
**Pull request by elgatovital**
Closes #370: Adjust transaction list headers

Changes proposed in this pull request:
- the header between entries correspond to the data (no grouping of multiple dates)

@gnosis/mobile-devs

### CODE REVIEW

File: app/src/test/java/pm/gnosis/heimdall/ui/safe/details/transactions/AdapterEntryHeaders.kt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package pm.gnosis.heimdall.ui.safe.details.transactions
+
+const val PENDING = "pending"
```
Comment (rmeissner) [inline app/src/test/java/pm/gnosis/heimdall/ui/safe/details/transactions/AdapterEntryHeaders.kt]: What are these used for? Are they only for testing?

Comment (elgatovital) [reply app/src/test/java/pm/gnosis/heimdall/ui/safe/details/transactions/AdapterEntryHeaders.kt]: Yes

Comment (rmeissner) [reply app/src/test/java/pm/gnosis/heimdall/ui/safe/details/transactions/AdapterEntryHeaders.kt]: I would move them to the test folder in this case, else it looks good

Comment (elgatovital) [reply app/src/test/java/pm/gnosis/heimdall/ui/safe/details/transactions/AdapterEntryHeaders.kt]: Ok

**Pull request merged**